python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -937,8 +938,13 @@ def _fast_slow_function_call(
             domain="cudf_pandas",
         ):
             slow_args, slow_kwargs = _slow_arg(args), _slow_arg(kwargs)
-            with disable_module_accelerator():
-                result = func(*slow_args, **slow_kwargs)
+            if args[0].__name__ == "read_pickle":
```

Comment:
`result = func(*slow_args, **slow_kwargs)` led to infinite recursion so I had to get the argument needed for `read_pickle`. Is there a better way to do this?

Reply:
Curious what is the part leading to recursion?

If the goal is to always just call pandas' `read_pickle`/`to_pickle`, it may make sense to manually define these method wrapper in `python/cudf/cudf/pandas/_wrappers/pandas.py`.